src/map/clif.c

```diff
@@ -18572,6 +18589,25 @@ void clif_dressroom_open(struct map_session_data *sd, int view)
 	WFIFOSET(fd,packet_len(0xa02));
 }
 
+void clif_cartdeco(struct map_session_data *sd)
+{
+#if PACKETVER >= 20150805
+	int i = 0, cart_deco = 3, fd;
```

Comment:
Could you explain the purpose of the `cart_deco` variable? (it doesn't look like it can be changed anywhere, is it always 3?)